src/lib.rs

```diff
@@ -136,4 +162,21 @@ mod tests {
 
         assert!(color.r + color.g + color.b != another_color.r + another_color.g + another_color.b);
     }
+
+    #[test]
+    fn generates_random_location() {
+        let mut utility = Utility::new();
+        let width = 100.0f32;
+        let height = 100.0f32;
```

Comment:
We don't need the `f32` explicit type casting. As we have the types set in the function signature Rust knows that these numbers must be f32's and therefore will only let us compile when the proper type. The `.0` part tells Rust this is an f32.